### PR TITLE
Remove gestures, paper-button, and fix imports

### PR DESCRIPTION
**Pull request by abdonrd**
Now should work 🤞 

Also, I indent the HTML like the other PolymerElements.

Easier to review without whitespaces: [#35/files?w=1](https://github.com/Polymer/lazy-imports/pull/35/files?w=1)

### CODE REVIEW

**Review by stramel — APPROVED** [review]

**Comment by stramel** [comment]
@abdonrd do you mind adding the imports in test/(1.x|2.0)/lazy-element and fixing the polymer import in 2.0 to use element? Also, maybe just go ahead and remove the paper-button dependency from the project by removing the last uses in test folder?

File: test/1.x/lazy-element.html [hunk 1]
```diff
@@ -7,24 +7,31 @@
...
       is: 'lazy-element-behavior',
+
       properties: {
         importLoaded: Boolean,
       },
+
       ready: function() {
         this.importLoaded = true;
-      },
+      }
```
Comment (stramel) [inline test/1.x/lazy-element.html]: Doesn't the comma need to be here for trailing comma eslint rule?

Comment (abdonrd) [reply test/1.x/lazy-element.html]: I don't know. Travis didn't say anything.

Comment (stramel) [reply test/1.x/lazy-element.html]: Okay, I just thought the Google preset required trailing comma. If not no worries

Comment (justinfagnani) [reply test/1.x/lazy-element.html]: we use trailing commas

Comment (abdonrd) [reply test/1.x/lazy-element.html]: Added!

File: test/1.x/index.html [hunk 2]
```diff
@@ -8,19 +7,23 @@
...
-    ]);
-  </script>
-</body>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>lazy-imports tests (Polymer 1.x)</title>
+
+    <script src="../../web-component-tester/browser.js"></script>
```
Comment (justinfagnani) [inline test/1.x/index.html]: did this URL change break or fix the test?

Comment (abdonrd) [reply test/1.x/index.html]: Oops! Fixed!

**Review by stramel — APPROVED** [review]
LGTM

**Comment by stramel** [comment]
Ping @rictic @justinfagnani

**Comment by stramel** [comment]
@abdonrd Pulling in `polymer/polymer-element.html` on the mixin page is giving me an error of `importHref` is not defined... May want to check this before getting merged.

**Comment by abdonrd** [comment]
@stramel pleae, try now! :)

**Comment by stramel** [comment]
@abdonrd Should those imports go into the mixin instead so we aren't adding them to the behavior as well? Behavior is importing the legacy-element which should include both of those.

**Comment by abdonrd** [comment]
@stramel humm... I'm not sure. Having the two versions together is a bit strange for me.

I think ideally we shouldn't use `Polymer.DomModule` and `Polymer.importHref` in `lazy-imports-import.html`. Because in this file, they don't exist. 😕

**Comment by stramel** [comment]
@abdonrd Yeah, downside of running a hybrid setup. I think the way you have it now is correct. I have it working in my testing repo now with this change. :+1:

**Review by stramel — APPROVED** [review]

**Comment by abdonrd** [comment]
Merged `master` branch!

**Comment by stramel** [comment]
@abdonrd Looks like you have a couple lint errors

**Review by stramel — APPROVED** [review]

**Pull request merged**